estate/models/estate_property_tag.py

```diff
@@ -0,0 +1,9 @@
+from odoo import fields, models
+
+
+class EstatePropertyTag(models.Model):
+    _name = 'estate.property.tag'
+    _description = 'Estate property tag model'
+
+
+    name = fields.Char('Title')
```

Comment:
```suggestion
    name = fields.Char(string='Title')
```
Follow this guideline and check other occurrences as well.

Reply:
Okay.